### ISSUE TITLE
0.23.15 Update

### ISSUE THREAD

**Issue by TinyMikeG** [FreeTubeApp/FreeTube#8827]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

Notified in 0.23.14  that there is a 0.23.15 update but the page still shows the current version. I downloaded it to confirm it is still 0.23.14. Is there an update or not?

### Expected Behavior

Available update

### Issue Labels

accessibility issue

### FreeTube Version

0.23.14

### Operating System Version

Ubuntu

### Installation Method

.deb

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Website is having some issues right now. We'll try to resolve that asap. In the mean time its best to download from GitHub

https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.23.15-beta

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Should be resolved now
